### ISSUE TITLE
[FEATURE] Automatically select `bedrock-latest` as version when creating a bedrock path

### ISSUE THREAD

**Issue by Pomi108** [Faithful-Resource-Pack/App#68]
Title speaks for itself. This would save a bit of time when creating paths. It would be nice to see it implemented both in the Add Texture modal as well as the editing one.

**Comment by Juknum** [comment]
Uh it would be good for now, but will cause issue for the dungeons edition ? Most of the path are the same ? (I haven't verified this)

**Comment by Juknum** [comment]
You mean trough the modal? When creating a path from a bedrock use ?

**Comment by Pomi108** [comment]
yeah, when the use edition is set to bedrock, the version should automatically be set to `bedrock-latest`
![obrazek](https://user-images.githubusercontent.com/59484948/174485794-f5bf4b59-24b3-4254-bc2f-7dfa61e18ead.png)
